evm/src/main/java/com/horizen/evm/interop/TracerConfig.java

```diff
@@ -1,23 +0,0 @@
-package com.horizen.evm.interop;
-
-import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
-
-@JsonIgnoreProperties(ignoreUnknown = true)
-public class TracerConfig {
```

Comment:
As I understood it is data-class specific to `callTracer` only.
So we now keep this field as a part of `TraceOptions` class in a free JsonNode form and delegate go part to validate and use it to configure a specific tracer.
So in case of any custom creation error:
```
if err != nil {
			log.Warn("failed to create tracer: %v", err)
		}
```
we will just ignore tracer the same as in Geth, correct?

Reply:
Good point. In case of an error we log a warning and continue without a tracer, Geth would throw an error. I will change this.

Currently, the only native tracer that has any arguments is the `callTracer`, correct. Theoretically though, they all can have different parameters, which is why we don't parse `TraceOptions.tracerConfig` anymore and just pass it along as raw JSON.

Reply:
done

Reply:
test case for an invalid tracer added to `sc_evm_test_debug_methods.py`